### DISCUSSION TITLE
Custom color for shapes and lines and numbers as well

### DISCUSSION THREAD

**Discussion by alamin5G** [pubpub-zz/ppInk#274] [Ideas]
Hello, it is possible to change the color from yellow to another color? Because, sometime, our screen are not matched with the yellow color, so, it is better to if you add customized color option, we will be pleased! 

Thank you so much, for your big effort. It is impossible to make it without your hard work and your valuable time.

**Comment by alamin5G** [comment]
Sorry dude, you already implemented it. I am very pleased!

**Reply by pubpub-zz** [reply]
Happy you are enjoying 😋
